### DISCUSSION TITLE
Enhancement - Filter for Today, Upcoming

### DISCUSSION THREAD

**Discussion by virgil9306** [NicolasPetton/Projectify#61] [Ideas]
Thanks for your work on this project.

So far it is the closest thing to my ideal task manager because it can be combined with the extensibility of Tiddlywiki there are so many possibilities.

I think that it would also be really helpful to have a simple string-based filter (based on fields) for hiding / showing items based on different projects, or date ranges (not just "Today" and "Upcoming" but custom date ranges).

(Closed #60 because I thought it is more appropriate here.)

**Comment by NicolasPetton** [comment]
@virgil9306 the Filter tab of the dashboard doesn't work for you?

**Comment by virgil9306** [comment]
Is there any way to apply the filter before the tab loads? On first view?

**Reply by NicolasPetton** [reply]
Not currently, no, but if you want custom date ranges, I'm not sure I understand how it would work.

**Comment by virgil9306** [comment]
I’m quite new to tiddlywiki as well, but I was thinking a “between dates X
and Y.”

But perhaps this could just be considered as a suggestion. My use case was
“see all the todos between X and Y dates,” not sure if others need that.

Thanks again for your response.
